src/js/components/Carousel/doc.js

```diff
@@ -23,6 +23,8 @@ export const doc = Carousel => {
     play: PropTypes.number.description(`If specified, the number of
       milliseconds between automatically transitioning to the next child. It
       will loop through all children indefinitely.`),
+    initialActiveIndex: PropTypes.number.description(`If specified, the index of
```

Comment:
I'm not excited about the prop name `initialActiveIndex`, I wish we could make it shorter/simpler, what about `initialChild`? it may be more consistent with the 'child' terminology used on the Carousel docs? Thoughts? @oorestisime @ericsoderberghp

Reply:
I have no strong arguments on the name, so whatever you pick is fine by me. Do you want to wait for @oorestisime and @ericsoderberghp or can I rename to `initialChild`?

Reply:
I renamed to initialChild 😬 makes more sense indeed.